src/core/stream_recv.c

```diff
@@ -257,6 +328,10 @@ QuicStreamProcessResetFrame(
             (void)QuicStreamIndicateEvent(Stream, &Event);
         }
 
+        QuicTraceLogStreamVerbose(
```

Comment:
This is already handled by the StreamRecvState event.

Reply:
I can remove!